docs/install/knative-with-operators.md

```diff
@@ -1,84 +1,74 @@
+Knative provides a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) to install, configure and manage Knative.
 
-## Before you begin
-
-Knative installation using the Operator requires the following:
-
-- A Kubernetes cluster v1.16 or newer, as well as a compatible kubectl. This guide assumes that you've already created
-a Kubernetes cluster. If you have only one node for your cluster, set CPUs to at least 6, Memory to at least 6.0 GB,
-Disk storage to at least 30 GB. If you have multiple nodes for your cluster, set CPUs to at least 2, Memory to at least
-4.0 GB, Disk storage to at least 20 GB for each node.
-- The Kubernetes cluster must be able to access the internet, since the Knative operator downloads images online.
-- [Download and install Istio](./installing-istio.md).
-
-## Limitations of Knative Operator:
-
-Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
+**NOTE:** The Knative Operator is still in Alpha phase. It has not been tested in a production environment, and should be used
 for development or test purposes only.
 
-## Install Knative with the Knative Operator
+## Prerequisites
 
-You can find the release information of Knative Operator on the [Releases page](https://github.com/knative/operator/releases).
+- You have a cluster that uses Kubernetes v1.17 or newer.
+- You have installed the `kubectl` CLI.
+- If you have only one node in your cluster, you will need at least 6 CPUs, 6 GB of memory, and 30 GB of disk storage.
+- If you have multiple nodes in your cluster, for each node you will need at least 2 CPUs, 4 GB of memory, and 20 GB of disk storage.
+- Your Kubernetes cluster must have access to the internet, since the Knative Operator must download images.
+- You have installed [Istio](./installing-istio.md).
 
-### Installing the Knative Operator
+## Installing the latest release
 
-__From releases__:
+You can find the release information about the Knative Operator on the [Releases page](https://github.com/knative/operator/releases).
 
-Install the latest Knative operator with the following command:
+To install the latest stable Operator release, enter:
 
 ```
 kubectl apply -f {{< artifact org="knative" repo="operator" file="operator.yaml" >}}
 ```
 
-__From source code__:
+## Installing from source
 
-You can also install Knative Operator from source using `ko`.
+You can install the Knative Operator from the source code using the [ko](https://github.com/google/ko) build tool.
 
-1. Install the [ko](https://github.com/google/ko) build tool.
-1. Download the source code using the following command:
+1. Download the source code:
 
-```
-git clone https://github.com/knative/operator.git
-```
+    ```
+    git clone https://github.com/knative/operator.git
+    ```
 
-1. Install the operator in the root directory of the source using the following command:
+1. Install the Operator in the root directory of the source:
```

Comment:
We lost the `ko` requirement and link for build-from-source, but I think we should probably also think about the cases that would lead to building from source in a non-contributor role.

Reply:
Leave building from source be an advanced topic in sync with github repo so the instructions should be in github serving both users that want to compile and user that want to contribute code and need to compile

Reply:
Removing in follow up PR